src/service/performance-impl.js

```diff
@@ -136,10 +133,7 @@ export class Performance {
     this.supportsLayoutShift_ = supportedEntryTypes.includes('layout-shift');
 
     if (!this.supportsLayoutShift_) {
-      this.metrics_.rejectSignal(
-        TickLabel.CUMULATIVE_LAYOUT_SHIFT,
-        new Error('Cumulative Layout Shift not supported')
```

Comment:
We need to pass an error here. A good idea is to follow @rcebulko's [suggestion](https://github.com/ampproject/amphtml/pull/29114#issuecomment-654926402) and use `expectedError`.